lib/matplotlib/tests/test_axes.py

```diff
@@ -3514,6 +3514,18 @@ def test_eb_line_zorder():
     ax.set_title("errorbar zorder test")
 
 
+@image_comparison(baseline_images=['axline'], extensions=['png'],
+                  style='mpl20')
+def test_axline():
```

Comment:
could use check_figures_equal instead.

Reply:
To do what? Currently this is a standalone figure tests with it's own figure (I think).

Reply:
I'd write the test as
```
@check_figures_equal()
def test_axline(fig_test, fig_ref):
    ax = fig_test.subplots()
    ax.set(xlim=(-1, 1), ylim=(-1, 1))
    ax.axline(1, 0)
    ax.axline(0, 0, color='C1')
    ax.axline(0, 0.5, color='C2')
    ax = fig_ref.subplots()
    ax.set(xlim=(-1, 1), ylim=(-1, 1))
    ax.plot([-1, 1], [-1, 1])
    ax.axhline(0, color='C1')
    ax.axhline(0.5, color='C2')
```
(and remove the baseline image).

Reply:
This comment still holds, modulo the API change (this avoids adding a baseline image to the repo, and I think it's semantically a clearer test too ("doing this plot should result in the same thing as doing that plot")).

Reply:
👍 will change it soonish (I'll mark as WIP in the meantime)